src/dbgshim/dbgshim.vcxproj.filters

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why do we need this filters?

Reply:
All the native projects that I added have the *.filters files. Are you saying that they are not needed?  I thought they define the project folders?

Reply:
Maybe that's what they do - I'd trust your knowledge of VS much more than mine.